bazel/repository_locations.bzl

```diff
@@ -1,8 +1,20 @@
+# Validation of content in this file is done on the bazel/repositories.bzl file to make it free of bazel
+# constructs. This is to allow this file to be loaded into Python based build and maintenance tools.
+
 # Envoy dependencies may be annotated with the following attributes:
-#
-# use_category - list of the categories describing how the dependency is being used. This attribute is used
-#                for automatic tracking of security posture of Envoy's dependencies.
-#                Possible values are documented in the USE_CATEGORIES list.
+DEPENDENCY_ANNOTATIONS = [
+    # List of the categories describing how the dependency is being used. This attribute is used
+    # for automatic tracking of security posture of Envoy's dependencies.
+    # Possible values are documented in the USE_CATEGORIES list below.
+    # This attribute is mandatory for each dependecy.
+    "use_category",
+
+    # Optional attribute specifying CPE (Common Platform Enumeration, see https://nvd.nist.gov/products/cpe) ID
```

Comment:
Can we make it compulsory? At least for certain use_categories, e.g. dataplane.

Reply:
Done. A lot of dependencies will have "N/A" as they have no CPEs assigned.